mobile-app/app/screens/AppNavigator/screens/Auctions/screens/ConfirmPlaceBidScreen.tsx

```diff
@@ -0,0 +1,210 @@
+import { hasTxQueued, transactionQueue } from '@store/transaction_queue'
+import { firstTransactionSelector, hasTxQueued as hasBroadcastQueued } from '@store/ocean'
+import { onTransactionBroadcast } from '@api/transaction/transaction_commands'
+import { CTransactionSegWit, PlaceAuctionBid } from '@defichain/jellyfish-transaction/dist'
+import { WhaleWalletAccount } from '@defichain/whale-api-wallet'
+import { NativeLoggingProps, useLogger } from '@shared-contexts/NativeLoggingProvider'
+import { SubmitButtonGroup } from '@components/SubmitButtonGroup'
+import { TextRow } from '@components/TextRow'
+import { NumberRow } from '@components/NumberRow'
+import { FeeInfoRow } from '@components/FeeInfoRow'
+import { ThemedScrollView, ThemedSectionTitle } from '@components/themed'
+import { AuctionsParamList } from '../AuctionNavigator'
+
+type Props = StackScreenProps<AuctionsParamList, 'ConfirmPlaceBidScreen'>
+
+export function ConfirmPlaceBidScreen (props: Props): JSX.Element {
+  const navigation = useNavigation<NavigationProp<AuctionsParamList>>()
+  const dispatch = useDispatch()
+  const logger = useLogger()
+  const hasPendingJob = useSelector((state: RootState) => hasTxQueued(state.transactionQueue))
+  const hasPendingBroadcastJob = useSelector((state: RootState) => hasBroadcastQueued(state.ocean))
+  const currentBroadcastJob = useSelector((state: RootState) => firstTransactionSelector(state.ocean))
+  const {
+      bidAmount,
+      estimatedFees,
+      totalAuctionValue,
+      vault,
+      batch
+     } = props.route.params
+
+  const [isSubmitting, setIsSubmitting] = useState(false)
+  const [isOnPage, setIsOnPage] = useState(true)
+
+  useEffect(() => {
+    setIsOnPage(true)
+    return () => {
+      setIsOnPage(false)
+    }
+  }, [])
+
+  async function onSubmit (): Promise<void> {
+    if (hasPendingJob || hasPendingBroadcastJob) {
+        return
+    }
+
+    setIsSubmitting(true)
+    await constructSignedBidAndSend(
+      vault.vaultId,
+      batch.index,
+      {
+        amount: bidAmount,
+        token: Number(batch.loan.id)
+      },
+      batch.loan.displaySymbol,
+      dispatch, () => {
+        onTransactionBroadcast(isOnPage, navigation.dispatch)
+    }, logger)
+    setIsSubmitting(false)
+  }
+
+  function onCancel (): void {
+    if (!isSubmitting) {
+      navigation.navigate({
+        name: 'PlaceBidScreen',
+        params: {
+          batch,
+          vault
+        },
+        merge: true
+      })
+    }
+  }
+
+  function getSubmitLabel (): string {
+    if (!hasPendingBroadcastJob && !hasPendingJob) {
+      return 'CONFIRM BID'
+    }
+    if (hasPendingBroadcastJob && currentBroadcastJob !== undefined && currentBroadcastJob.submitButtonLabel !== undefined) {
+      return currentBroadcastJob.submitButtonLabel
+    }
+    return 'PLACING BID'
+  }
+
+  return (
+    <ThemedScrollView
+      testID='confirm_place_bid_screen'
+      contentContainerStyle={tailwind('py-6 pb-8 px-4')}
```

Comment:
Let's remove the unnecessary padding at the side and top
![image](https://user-images.githubusercontent.com/40191153/144533758-61c17501-3f3a-4a08-af0f-40b76fab7bc7.png)